scipy/stats/morestats.py

```diff
@@ -1453,11 +1455,18 @@ def rootfunc(ab, xj, N):
         logsf = distributions.logistic.logsf(w)
         sig = array([25, 10, 5, 2.5, 1, 0.5])
         critical = around(_Avals_logistic / (1.0 + 0.25/N), 3)
-    else:  # (dist == 'gumbel') or (dist == 'extreme1'):
-        xbar, s = distributions.gumbel_l.fit(x)
-        w = (y - xbar) / s
-        logcdf = distributions.gumbel_l.logcdf(w)
-        logsf = distributions.gumbel_l.logsf(w)
+    else:  # ((dist == 'gumbel') or (dist == 'gumbel_r') or
+        # (dist == 'gumbel_l') or (dist == 'extreme1')):
+        if dist == 'gumbel_r':
```

Comment:
Any reason this is not at the same level as `if dist == "norm"... elif dist == "expon" ... elif ...`?